kv-storage/interface.https.html

```diff
@@ -140,29 +140,30 @@
 
   await frameLoadPromise(iframe);
   const OtherStorageArea = iframe.contentWindow.StorageArea;
+  const TypeError = ifrae.contentWindow.TypeError;
 
-  await promise_rejects(t, new TypeError(),
+  await promise_rejects_js(t, TypeError,
     OtherStorageArea.prototype.set.call(storage, "testkey", "testvalue"),
     `set() must reject cross-realm`);
-  await promise_rejects(t, new TypeError(),
+  await promise_rejects_js(t, TypeError),
```

Comment:
```suggestion
  await promise_rejects_js(t, TypeError,
```

Reply:
Yikes, yes.  Also, the definition of TypeError has another typo; need to s/ifrae/iframe/.  Will push a fixup for that.